scss/responsive.scss

```diff
@@ -0,0 +1,62 @@
+@import 'variables';
+
+@media screen and (max-width: 700px){
+
+  .navbar-text {
+    visibility: hidden;
+  }
+
+  .card-container {
+    display: grid;
+    grid-template-columns: repeat(1, max-content);
+    justify-content: center;
+    align-items: center;
+    max-width: 1200px;
+    width: 100%;
+    margin: 50px auto;
+  }
+
+  .background-card{
+    display: flex;
+    justify-content: space-around;
+    border-bottom: 1px solid $green;
+    flex-direction: column;
+    font-size: 14px;
+    height: 370px;
+    width: 220px;
+    position: relative;
+    margin-left: 20px;
+    z-index: 1;
+  }
+
+  .main-tech {
+    border-botom: 1px solid $green;
+    border-right: 0;
+    grid-row: 1/3;
+    height: 100%;
+    justify-content: center;
+  }
+
+  .title {
+
+    padding: 195px 30px 30px 150px;
+    margin-left: -20px;
+    font-size: $font-big;
+    position: relative;
+    z-index: 1;
+
+    &::after{
+      background-image: url(assets/triangle.svg);
+      background-size: 90px 90px;
+      position: absolute;
+      z-index: -1;
+      width: 90px;
+      height: 90px;
+      bottom: 5px;
+      left: 90px;
+      content:'';
```

Comment:
By only changing the position of the title it's enough to move this element too :smile: